extensions/json-schema/services/src/main/resources/META-INF/cxs/schemas/customitem.json

```diff
@@ -1,6 +1,12 @@
 {
   "$id": "https://unomi.apache.org/schemas/json/customitem/1-0-0",
   "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "self":{
+    "vendor":"The Apache Software Foundation",
+    "name": "customItem",
+    "format":"jsonschema",
+    "version":"1-0-0"
+  },
```

Comment:
Not sure this schema is still necessary, I dont see why event should use customItem for now.